public/docs/ts/latest/guide/architecture.jade

```diff
@@ -340,19 +326,18 @@ figure
   as with event binding.
 
   Angular processes *all* data bindings once per JavaScript event cycle,
-  from the root of the application component tree down to the leaves.
+  from the root of the application component tree through all components.
```

Comment:
How about: "from the root of the application component tree through all child components."? Not sure on how to incorporate "directly or indirectly" differently, though I like that wording, even if it does add length.